### DISCUSSION TITLE
text input chat client with mycroft problems

### DISCUSSION THREAD

**Discussion by FD3000** [community/community#109929] [New to GitHub]
Hello,

the title tells the most.. At this moment I don't have appropriate microphone and secondly, I would like to use mycroft by text input - instead of voice... somehow I haven't yet managed in this task... Can you help me please? :)

all the best

**Comment by ThisIsMrIsmail** [comment]
Hello @FD3000,

May you try the following steps I think it should work.

Check out skills specifically designed for text input. Search the Mycroft Skills Marketplace ([invalid URL removed]) for "text input" or "chat" related skills.

Some Mycroft setups might offer a web interface where you can type your commands. Search for documentation on how to access this on your device.

Mycroft often uses a messaging system for communication. You might be able to send messages directly to it as if it were another chat user. Check your Mycroft configuration for details.

To be extra helpful, could you tell me:
**Your Mycroft Device:** Are you using a Mark 1, Picroft, or something else?
**Your Setup:** Did you install Mycroft yourself, or is it part of a pre-made device?

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
